caddyfile_authn_ui.go

```diff
@@ -131,7 +132,7 @@ func parseCaddyfileAuthPortalUI(h *caddyfile.Dispenser, repl *caddy.Replacer, po
 				portal.UI.CustomJsPath = strings.ReplaceAll(args, "js ", "")
 			case strings.HasPrefix(args, "html header path"):
 				args = strings.ReplaceAll(args, "html header path ", "")
-				b, err := ioutil.ReadFile(args)
+				b, err := os.ReadFile(args)
```

Comment:
Go made me do it? `ioutil.ReadFile` is deprecated and apparently it just uses `os.ReadFile`